### ISSUE TITLE
Problem 2 - Tracking Judicial Findings of Police Misconduct

### ISSUE THREAD

**Issue by KallieFerguson** [Call-for-Code/Embrace-Judicial-Reform#21]
**Theme:** Police and Judicial Reform and Accountability

**Brief description of your idea:** Every day, in every county across the country, judges are issuing orders finding that police officers have broken the law--that they have lied on duty or on the stand, conducted illegal searches, used excessive force, or violated someone's civil rights in other ways. These orders are referred to as "suppression orders." 

Here is the problem: there is currently no organized way to track when an officer is subject to one of these orders.  That means prosecutors can continue to charge cases submitted by "bad" cops and tell criminal defense attorneys that they don't know whether the officer has ever been the subject of a suppression order.  It also means that officers may be able to testify in court--to a judge or a jury--without anyone knowing that they have previously broken the law while doing their jobs.  In fact, it's possible that the officers themselves (and accordingly, their supervisors) don't even know whether a judge has ever found that they have broken the law. Currently, officers fired in one county or state for misconduct can be hired as an officer in another county or state, without the new agency ever being aware that a judge has found the officer lied or engaged in misconduct.  (This happens all too frequently.). 

There is an easy fix: a nationwide user-generated database that contains information about these suppression orders and allows users to search the database by an officer’s name, jurisdiction, or department.  A database that tracks these suppression orders will help identify officers who are repeat offenders, important trends, and precincts that have troubling practices.  

**What makes your idea unique?:** 
First: 

An attorney who gets a suppression order from a judge would visit the website, fill out a form with information about the officer at issue, and immediately populate a searchable database.  When another attorney is preparing a case with an officer, s/he would go to the database and pull up every order issued about that officer.  That information goes directly to the officer's credibility when they testifies in court to a judge or jury.  It would also allow reporters and community organizers to see which officers in their communities are repeat offenders and should be terminated.

Second:

These orders are a quick and objective way to hold police officers, their departments, and prosecutors accountable. Unlike police complaints, which are private, and the internal investigations done on those complaints, which are private and typically biased, suppression orders are public.  They are also objective in that they are a decision from a judge that an officer violated the law.  The decision by the judge is usually the final word; not even a police sergeant or police union representative can argue with the finding or rationalize it away.

Third:

The database already exists in a simple format that can be improved dramatically with help.

With the assistance of a few tech-savvy friends, a former public defender and now civil rights attorney in Minneapolis created a website using Wix that hosts a database to track suppression orders and allows user to upload information using a form.  The website is https://www.thesuppressionordersproject.org/ and was created in the past two weeks.  The attorney is currently reaching out to different public defender officers and criminal defense attorneys around the country to explain its purpose and get people using it.  But with only 10 entries currently, the database is already slow.  To be effective, the website needs to be better and more user-friendly.  Ideally, there could be a username/password requirement, better search capability, and an app or mobile version that would quickly allow users to input or search information.

**What would be the impact of your idea if implemented?:** The very direct impact of the idea is this: police officers who repeatedly break the law would be fired, and police officers would know they would be held accountable for their actions.  As stated above, the database will also help identify important trends and precincts that boast troubling practices.

**Skills to contribute (e.g. development, architecture, research, design or anything else):** Even though the current version of the website exists on the Wix platform, development, architecture, and design are all skills that would be useful to improving the website and expanding its capacity. Research assistance is also required to compile contact information of the criminal defense groups in all 50 states to spread the word about the database.

Some ideas for improving the website include a document store with an Elasticsearch-driven UI and some basic tools for spam filtering, record verification marking, and user management with three roles (anonymous, user, and admin) and a submission form that's access controlled for logged in users (auth-0 social oAuth would work). Devs and product can play in the query-ui and data science space - chart reports on a map, predict likelihood a new record will be verified as accurate, user saved/share-able queries, etc. Being able to track a report from submission to verification and "appeared in x number of queries run."

**Comment by fjoenichols** [comment]
I was considering something along these lines with user reported misconduct reports, but using suppression orders as something that is impartial is a good idea. It could even be expanded to allow police departments to opt in to reporting submitted police reports or disciplinary actions.

**Comment by KallieFerguson** [comment]
Yes, I know I'm biased because this is my project idea, but I like that: 

1) it's an immediately useful tool
2) It's doable in the relatively short timeframe we have
3) It's using public data from court systems, so there are no data privacy concerns
4) It's a good starting point for objective findings of misconduct. 
5) Nothing like it exists currently 

Ultimately, it doesn't go far enough to prevent all misconduct, but analyzing for more subtle data requires a lot of public record requests, data that isn't publicly available, and a lot of nuance in the time frame that we have to build something.

**Comment by henrynash** [comment]
Some random ideas:
- Should we include positive mentions in court records, as well as the negative ones (i.e. suppression orders)?
- A view of any officer that is a bit like a credit rating (i.e. these 5 positives, and these 3 negatives)...giving average rating 456 out of 999.
- Should be able to view by Police Department as well as individual officer.  Maybe by state as well?
- Should we attempt to do an AI analysis of published court records? (i.e. to extract the above?) (if we feel this might be too risky, then maybe this could generate a pipeline of auto-created records that needed to be approved to be added)

**Comment by KallieFerguson** [comment]
> Some random ideas:
> 
> * Should we include positive mentions in court records, as well as the negative ones (i.e. suppression orders)?
> * A view of any officer that is a bit like a credit rating (i.e. these 5 positives, and these 3 negatives)...giving average rating 456 out of 999.
> * Should be able to view by Police Department as well as individual officer.  Maybe by state as well?
> * Should we attempt to do an AI analysis of published court records? (i.e. to extract the above?) (if we feel this might be too risky, then maybe this could generate a pipeline of auto-created records that needed to be approved to be added)

Hi Henry! 

Officers conduct is presumably positive and there are no special findings made by the court in that regard. So unless there's an order saying an officer engaged in misconduct, the officer is presumed to not have any misconduct. 

Adding state and police department search capabilities would be great. Would make it easier to see if specific states or departments have wide spread problems. 

I am not sure about the AI analysis. We may need to talk through the feasibility and what that would look like.

**Comment by henrynash** [comment]
@KallieFerguson Yes, I wasn't sure if there were court commendations etc...was just trying to think of a way where including positive aspects might make this seem more balanced - and harder for police officers/unions etc. to be against it...but maybe there just isn't a good enough (independent) source for that....

**Comment by henrynash** [comment]
(thinking/typing out loud here)....I believe one of the real problems is that (since there is no such database like this in common use yet) it is relatively easy for an officer with, say, multiple suspension orders to leave one department and join another. Given that I assume police badge numbers are unique to each police department, would the court records be enough to allow us to tie, say, "Officer Dibble" in dept A with "Officer Dibble" in dept B? I didn't; for instance, see SSN listed in the  https://www.thesuppressionordersproject.org/ database

**Comment by KallieFerguson** [comment]
@henrynash that's an issue for sure, because badge numbers would change. But currently, we would not be able to add more personal identifiable information than what is contained in the public court record (and would be opening ourselves up to legal issues if we did).

**Comment by henrynash** [comment]
@KallieFerguson agreed, that was my fear

**Comment by pjennas** [comment]
There's a nice rudimentary base here with the database for us to start from.  We do need to figure out a way to provide some form of context...which I think is what Henry was after with the idea of including positive info as well.  Since we don't have that then perhaps something like a color or level indicator e.g. I'm thinking an officer with a single incident of conducting an illegal search should be viewed differently than one with multiple incidents of planting evidence and committing assault so the former might have a yellow or "Level 1" indicator whereas the latter might be red or level 10.  Lawyers and judges may not have as much use for this since they'll want to review the details for a specific officer they're working with but given this is also avail to the public, the differentiation will be important for the public perception of an individual officer as well as help with quickly identifying the most serious offenders.

**Comment by KallieFerguson** [comment]
@pjennas that would be useful, although it will also require netting out the varying degrees of misconduct and violation levels with clear guidelines, which could get very nuanced very quickly. Possibly we could just build in the capability of assessing levels without defining them ourselves.

**Comment by pjennas** [comment]
@KallieFerguson perfect, that's exactly what I was thinking.  Don't know that any of us have the expertise to accurately define or rank the severity but build it in now, perhaps have our own mockup rankings just for demo purposes, and make them easily configurable by whatever entity ultimately owns the db.

**Comment by henrynash** [comment]
So my proposal would be that since we will be separating the components as part of the design, we keep the DB as "the reported facts", and enable the UI layer (or other sever-side APIs) to generate any such interpretations of the data.
